plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -117,14 +118,45 @@ public function is_complete() {
 			$third_party_complete = apply_filters( 'woocommerce_admin_third_party_tax_setup_complete', false );
 
 			$this->is_complete_result = $is_wc_connect_taxes_enabled ||
-				count( TaxDataStore::get_taxes( array() ) ) > 0 ||
-				get_option( 'woocommerce_no_sales_tax' ) !== false ||
-				$third_party_complete;
+                   get_option( 'woocommerce_no_sales_tax' ) !== false ||
```

Comment:
Just double checking, but seems like we're putting the less expensive checks first so we might not need to run the more expensive checks, right?

If so, 👍